tests/TestCase/Core/StaticConfigTraitTest.php

```diff
@@ -17,53 +17,6 @@
 use Cake\TestSuite\TestCase;
 
 /**
- * TestConnectionManagerStaticConfig
```

Comment:
Moving the tests let me remove the stub class which had code duplicated from `ConnectionManager`.